pyproject.toml

```diff
@@ -88,7 +89,7 @@ update_cli_docs = "python ./scripts/generate_reference_docs.py"
 shell = "pytest -n auto tests/e2e"
 
 [tool.poe.tasks.test_integration]
-shell = "pytest -n auto scripts tests/integration"
+shell = "pytest -n auto --random-order scripts tests/integration"
```

Comment:
It is not related to the PR

Reply:
It is. WIthout it, the problem with devnet/CI occurs. The diff should disappear after rebasing.

Reply:
It should have been fixed in a separate PR and then rebased